templates/results.html

```diff
@@ -1,5 +1,50 @@
-<p>Results are here:</p>
-<!-- Parādām mūsu faila rindas -->
-{% for value in lines %}
-  <p>{{ value }}</p>
-{% endfor %}
+<!DOCTYPE html>
```

Comment:
Es šo te nedaudz izmainīju.

Izmantoju to, par ko runāju apmācībās.

Soļi:
- Izveidoju jaunu layout.html failu, kur tad būs visi mūsu javascript un css faili
saturs:
```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset='UTF-8'>
    <title>21</title>
    <script src="..."></script>
    <link rel="stylesheet" href="..">
  </head>
  <body>
    {% block content %}
    {% endblock %}
  </body>
</html>
```
- tad pārtaisīju mūsu index.html failu, kas tagad izskatās šādi:
```html
{% extends 'layout.html' %}
{% block content %}
  <div id='wrapper'>
    <div id='game'>
      <div id='dealer-wrapper'>
        <div id='dealer-chip'>Dealer</div>
        <div id='dealer' class='hand'></div>
        <div id='in-game-deck'><p id='in-game-deck-points'></p></div>
      </div>
      <div class='actions'>
        <div id='before-game-actions'>
          <button onClick='table.startGame()'>Start</button>
        </div>
        <div id='after-game-actions'>
          <button onClick='table.hit()'>Hit</button>
          <button class="double" onClick='table.double()'>Double</button>
          <button onClick='table.stand()'>Stand</button>
        </div>
        <div id='player-chip'>Play</div>
      </div>
      <div id='player' class='hand'></div>
    </div>
  </div>
{% endblock %}
```
- un beigās rezultāti:
```html
{% extends 'layout.html' %}
{% block content %}
  <div class="container mb-3 mt-3">
    <table id="table" class="table table-striped table-bordered w-100">
      <thead>
          <tr>
            <th>Name</th>
            <th>Money</th>
            <th>Status</th>
          </tr>
      </thead>
      <tbody>
          {% for line in lines %}
          {% set items = line.split() %}
            <tr>
              <td>{{ items[0] }}</td>
              <td>{{ items[1] }}</td>
              <td>{{ items[2] }}</td>
            </tr>
          {% endfor %}
      </tbody>
    </table>
  </div>
{% endblock %}
```

Tā man liekas, ka ir pārskatāmāk un korektāk organizēt kodu, mazāk dublējās un viss ir tikai vienā vietā!